cypress/e2e/po/components/sortable-table.po.ts

```diff
@@ -115,4 +115,12 @@ export default class SortableTablePo extends ComponentPo {
   selectAllCheckbox(): CheckboxInputPo {
     return new CheckboxInputPo('[data-testid="sortable-table_check_select_all"]');
   }
+
+  selectedCount() {
+    return cy.get('.sortable-table tbody input[type="checkbox"]:checked').its('length');
```

Comment:
`cy.self().find('.row-check input[type="checkbox"]:checked)` will be safer
- looks for results in this specific table
- looks for check boxes in what should be the specific select column